packages/stark-ui/src/modules/progress-indicator/directives/progress-indicator.directive.ts

```diff
@@ -49,13 +57,13 @@ const directiveName: string = "[starkProgressIndicator]";
 })
 export class StarkProgressIndicatorDirective implements OnInit, OnDestroy {
 	@Input()
-	public starkProgressIndicator: StarkProgressIndicatorConfig;
+	public starkProgressIndicator: StarkProgressIndicatorConfig = DEFAULT_PROGRESS_INDICATOR_CONFIG;
```

Comment:
I don't think defining a default value here is a good solution because it has a side effect: an instance is registered via the `StarkProgressIndicatorService`. This means that for every instance of this directive a new config with topic "undefined" will be registered :(

Reply:
Maybe it's better to make it a definite assignment and add 
```
if (!this.starkProgressIndicator) {
			throw new Error("[starkProgressIndicator] is required when using the starkProgressIndicator directive ");
		}
```
to the onInit?

Reply:
Yes, maybe that solution is better 👍